### ISSUE TITLE
Commands not working in new servers added while bot was running

### ISSUE THREAD

**Issue by RillJ** [Gitfoe/ChadCounting#2]
Bot seems to be stuck up on [this error](https://imgur.com/OMGfFx8) which occurred in command /setchannel.

**Comment by RillJ** [comment]
Issue seems to be that when a new guild gets added while the bot is running, it tries to add the whole `guild` and not just the `guild id`.
```
    yield from _iterencode_dict(o, _current_indent_level)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.496.0_x64__qbz5n2kfra8p0\Lib\json\encoder.py", line 377, in _iterencode_dict
    raise TypeError(f'keys must be str, int, float, bool or None, '
TypeError: keys must be str, int, float, bool or None, not Guild
```

**Comment by RillJ** [comment]
In the above case, the json file got corrupted because it wrote data that was not serializable data (in this case, a full guild Discord object instead of a string). Every time the program tried to write json data, it errored out at the point in the guild_data dictionary that tried to write the guild Discord object.

**Comment by RillJ** [comment]
Fixed in [4285aa8](https://github.com/Gitfoe/ChadCounting/commit/4285aa826ee46c743c77949fabee1f4f7044cecd)
